pkg/config/containers.conf

```diff
@@ -262,7 +262,13 @@ default_sysctls = [
 
 # Path to directory where CNI plugin binaries are located.
 #
-#cni_plugin_dirs = ["/usr/libexec/cni"]
+#cni_plugin_dirs = [
+#  "/usr/local/libexec/cni",
+#  "/usr/local/lib/cni",
+#  "/usr/libexec/cni",
```

Comment:
I believe we should leave /usr/libexec/cni first.
Also this list should appear in containers.conf.5.md

Reply:
Delete /usr/libexec/cni?

Reply:
No, I don't want to see the search order changed on an update.  This could be a breaking change.

If cni is working fine and then I update containers-common, suddenly I am using a different cni_plugins_dirs.  If we want to add the new defaults, we should add them after the old default.

Reply:
All new paths added after it?

Reply:
Yes

Reply:
```
/usr/libexec/cni
/usr/lib/cni
/usr/local/libexec/cni
/usr/local/lib/cni
```

Like this?

Reply:
Go with:

/usr/libexec/cni
/usr/local/libexec/cni
/usr/local/lib/cni
/usr/lib/cni

Reply:
Is there a chance to advance the path starting with /usr/local in the future? This would make it easier for the administrator to put the modified cni in /usr/local